web/src/invite.ts

```diff
@@ -226,7 +234,7 @@ function submit_invitation_form(): void {
                 .safeParse(xhr.responseJSON);
             if (!parsed.success) {
                 // There was a fatal error, no partial processing occurred.
-                ui_report.error("", xhr, $invite_status);
+                ui_report.error($t({defaultMessage: "Failed"}), xhr, $invite_status);
```

Comment:
I guess is there a future TODO to make `$invite_status` use a banner component as well? Just noticed we replaced the success code path, but not the error code path.